patches/react-native-touch-id+4.3.0.patch

```diff
@@ -0,0 +1,38 @@
+patch-package
+--- a/node_modules/react-native-touch-id/TouchID.m
++++ b/node_modules/react-native-touch-id/TouchID.m
+@@ -14,12 +14,13 @@ RCT_EXPORT_METHOD(isSupported: (RCTResponseSenderBlock)callback)
+     if ([context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:&error]) {
```

Comment:
who's handling these errors? the the touchid library of should we handle them in out code?

Reply:
The handling of the error is done in the library js code

https://github.com/naoufal/react-native-touch-id/blob/master/TouchID.ios.js#L39